README.md

```diff
@@ -4,22 +4,38 @@ This code simulates proton-electron collisions using Pythia8 and saves in HepMC
 
 ## Installation
 
-To run this code, you need to install [Pythia8](https://pythia.org/), a program for the generation of high-energy physics collision events. You can download Pythia8 using:
+1) Get and run [eic-shell](https://eic.github.io/tutorial-setting-up-environment/02-eic-shell/index.html)
+
+2) Clone and make Pythia8.306
 ```
-git clone git@gitlab.com:Pythia8/releases.git --branch pythia8XXX --single-branch
+mkdir Pythia8306
+cd Pythia8306
+git clone https://gitlab.com/Pythia8/releases.git --branch pythia8306 --single-branch
```

Comment:
Can we use the default branch here? And do we need `--single-branch`? More git options make this less understandable to those who don't know what the options mean.

Reply:
I put pythia8306 here because that's what the datasets were produced with in the first release. Probably don't need the single branch option.